app/api/entities/entities.js

```diff
@@ -365,7 +366,11 @@ const withDocuments = async (entities, documentsFullText) => {
       ? idFileMap.get(entity.sharedId).map(file => ({ ...file }))
       : [];
     entity.documents = entityFiles.filter(f => f.type === 'document');
-    entity.attachments = entityFiles.filter(f => f.type === 'attachment');
+    entity.attachments = orderBy(
```

Comment:
We had to introduce a sorting here because we couldn't handle it in a proper way at the edition of an entity on the client. We know this could be seen as a client coupling but still, we think it's not so bad that the API returns the attachments with a default order instead of a random one.